src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -139,6 +145,21 @@ protected static List<Configuration> getModuleConfigsByIds(String... moduleIds)
         return getModuleConfigsByIds(CONFIGURATION, moduleIds);
     }
 
+  /**
+   * Formats the file with Google Java Format.
+   *
+   * @param filePath file path to format.
+   * @throws IOException if an I/O error occurs.
+   * @throws FormatterException if an error occurs during formatting.
+   */
+    protected void formatFileWithGoogleJavaFormat(String filePath)
+            throws IOException, FormatterException {
+        final Path path = Paths.get(filePath);
+        final String original = Files.readString(path);
+        final String formatted = new Formatter().formatSource(original);
+        Files.write(path, formatted.getBytes(StandardCharsets.UTF_8));
+    }
+
```

Comment:
Created this method which will format the given file with google-java-format. This will be used in `whole config` tests.